BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+# Manage BigCommerce connection
```

Comment:
Denke bitte daran, noch die Related content Links in Integration, Troubleshooting and User interface Datei einzufügen.

Reply:
done